persistence-modules/spring-data-jpa-annotations/src/main/java/com/baeldung/attribute/overwrite/entity/Car.java

```diff
@@ -0,0 +1,51 @@
+package com.baeldung.attribute.overwrite.entity;
+
+
+import javax.persistence.*;
+import java.util.Map;
+
+@Entity
+@AttributeOverride(name = "identifier", column = @Column(name = "VIN"))
+public class Car extends Vehicle {
+
+    private String model;
+    private String name;
+    @Embedded
+    @AttributeOverrides({
+            @AttributeOverride(name = "name", column = @Column(name = "BRAND_NAME", length = 5)),
```

Comment:
Line continuations should use a two space indent

Reply:
Done.